sdk/storage/Azure.Storage.Files.Shares/src/autorest.md

```diff
@@ -4,7 +4,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://github.com/Azure/azure-rest-api-specs/blob/f23b5240a97736f1f04bfe49fba5a393903457f1/specification/storage/data-plane/Microsoft.FileStorage/preview/2021-06-08/file.json
+    - C:\Users\seanmcc\git\azure-rest-api-specs\specification\storage\data-plane\Microsoft.FileStorage\preview\2021-06-08\file.json
```

Comment:
TODO change this after https://github.com/Azure/azure-rest-api-specs/pull/18054 merges